test/expectRequestHelpers.js

```diff
@@ -0,0 +1,34 @@
+const chai = require('./chai');
+
+const { expect } = chai;
+
+function expectJSONResponse(res) {
```

Comment:
nit: wonder if these custom checks should be plugged into chai:

https://www.chaijs.com/api/plugins/

Reply:
:+1: It would be nicer than the way that `supertest` works now that promises are first class citizens in JS-land.  I haven't built any custom chai plugins before, so I did it as a standalone function for now in the interest of time...